src/ChipInput.js

```diff
@@ -635,13 +715,25 @@ ChipInput.defaultProps = {
 
 export default withStyles(styles)(ChipInput)
 
-export const defaultChipRenderer = ({ value, text, isFocused, isDisabled, handleClick, handleDelete, className }, key) => (
+export const defaultChipRenderer = ({ value, text, isFocused, isDisabled, handleClick, handleDelete, className, color }, key) => (
   <Chip
     key={key}
     className={className}
     style={{ pointerEvents: isDisabled ? 'none' : undefined, backgroundColor: isFocused ? blue[300] : undefined }}
     onClick={handleClick}
     onDelete={handleDelete}
     label={text}
+    color={color || 'default'}
   />
 )
+
+export const defaultInputAdornmentRenderer = (inputAdornment, additionalClasses) => {
```

Comment:
Not using this function would probably break a lot of things. Is a prop for customizing the renderer actually needed ? :thinking:

Reply:
The biggest item for this would be to resolve the classes for styling. It was loosely based on what was happening from any of the inputs in Material-ui compared to BaseInput.

When you say not using this function, can you clarify what issue you are seeing? It is a default renderer function and if someone were to override it, I would be expecting them to resolve the styling issues in their manual implementation anyhow. I was opting for customization capability more than anything.

Reply:
But I could easily just hide this from being a prop if that is what you are going after. Let me know if you want me to do that instead